src/cookies.js

```diff
@@ -183,7 +183,7 @@ export function getHighestAvailableDomain(win) {
  * @param {time} expirationTime
  * @param {string|undefined} domain
  * @param {!SameSite|undefined} sameSite
- * @param {boolean=} secure
+ * @param {boolean} secure
```

Comment:
This can be undefined, too.

Reply:
Lines 157 and 160 call `trySetCookie` without the last two args :(